graviti/exception.py

```diff
@@ -27,6 +27,52 @@ def __str__(self) -> str:
         return self._message if self._message else ""
 
 
+class PortexError(GravitiException):
+    """This is the base class for custom exceptions in Graviti portex module."""
+
+
+class GitNotFoundError(PortexError):
+    """This class defines the exception for the git command not found error.
+
+    Arguments:
+       message: The error message.
+       file_not_found_error: The FileNotFoundError raised from the subprocess.run().
+
+    """
+
+    _MESSAGE = (
+        "'git' command failed, most likely due to the 'git' is not installed.\n"
+        "Please install 'git' first: https://git-scm.com/book/en/v2/Getting-Started-Installing-Git"
+    )
+
+    def __init__(self, message: str = _MESSAGE):
```

Comment:
Should we add a class attr default message to `GravitiException` and rewrite its init method?

Reply:
actually I think we can remove the `message` in `GravitiException`

Reply:
Maybe we can add a `MessageMixin` to handle message.